contracts/pangolin-core/PangolinPair.sol

```diff
@@ -141,6 +141,7 @@ contract PangolinPair is IPangolinPair, PangolinERC20 {
 
         bool feeOn = _mintFee(_reserve0, _reserve1);
         uint _totalSupply = totalSupply; // gas savings, must be defined here since totalSupply can update in _mintFee
+
```

Comment:
this was actually not intentional, marking here so I revert when I get the chance